modules/andes-core/broker/src/main/java/org/wso2/andes/server/connection/ConnectionRegistry.java

```diff
@@ -38,22 +38,28 @@ public void initialise()
         // None required
     }
 
-    /** Close all of the currently open connections. */
-    public void close()
-    {
+    /**
+     * Close all existing client connections.
+     */
+    public void close() {
+        close("Broker is shutting down");
```

Comment:
we need to come up with a better message, right?

Reply:
Yes. The message we currently does not match the real reason. Will do the required change in a future PR.